### PR TITLE
【Hackathon 5th No.4】为 Paddle 新增 masked_scatter API 

### PR DESCRIPTION
**Pull request by yangguohao**
Add paddle.masked_scatter API cn doc

https://github.com/PaddlePaddle/Paddle/pull/59383

### CODE REVIEW

**Comment by yangguohao** [comment]
@luotao1

File: docs/api/paddle/masked_scatter_cn.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+返回一个 N-D 的 Tensor，Tensor 的值是根据 ``mask`` 信息，将 ``value`` 中的值逐个填充到 ``x`` 中 ``mask`` 对应为 ``True`` 的位置，``mask`` 的数据类型是 bool。
+
+参数
+::::::::::::
+
+    - **x** (Tensor) - 输入 Tensor，数据类型为 float，double，int，int64_t，float16 或者 bfloat16。
+    - **mask** (Tensor) - 布尔张量，表示要填充的位置。mask 的数据类型必须为 bool。
+    - **value** (Tensor)：用于填充目标张量的值，数据类型为 float，double，int，int64_t，float16 或者 bfloat16。
```
Comment (sunzhongkai588) [inline docs/api/paddle/masked_scatter_cn.rst]:
```suggestion
    - **value** (Tensor) - 用于填充目标张量的值，数据类型为 float，double，int，int64_t，float16 或者 bfloat16。
```

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 2]
```diff
@@ -625,7 +625,7 @@
 梳理了`torch.Tensor.XX`类 API 的 PyTorch-PaddlePaddle API 映射列表。
 
 | 序号  | PyTorch API | PaddlePaddle API | 备注 |
- |-----| --- | --- | --- |
+-----|-----| --- | --- | --- |
```
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md]: 为啥要改这个？

Comment (yangguohao) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md]: 恢复了

File: docs/api/paddle/masked_scatter_cn.rst (hunk 1 above)
Comment (yangguohao) [reply docs/api/paddle/masked_scatter_cn.rst]: done

File: docs/api/paddle/masked_scatter__cn.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.. _cn_api_paddle_masked_scatter_:
+
+masked_scatter\_
+-------------------------------
+
+.. py:function:: paddle.masked_scatter_(x)
```
Comment (sunzhongkai588) [inline docs/api/paddle/masked_scatter__cn.rst]:
```suggestion
.. py:function:: paddle.masked_scatter_(x, mask, value, name=None)
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.masked_scatter.md
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.masked_scatter.md:1]: 辛苦 @zhwesky2010 review下～

File: docs/api/paddle/masked_scatter__cn.rst (hunk 3 above)
Comment (yangguohao) [reply docs/api/paddle/masked_scatter__cn.rst]: done

**Review by zhwesky2010 — APPROVED** [review]

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**